.github/workflows/checks.yml

```diff
@@ -26,18 +28,18 @@ jobs:
       - name: Format
         run: npm run format
 
-      - name: Commit Formatting changes - will not trigger rebuild
+      - name: Commit Formatting changes
```

Comment:
I am not sure what was meant by the comment "will not trigger rebuild" but this action will now trigger CI.